doc/lightning-listchannels.7.txt

```diff
@@ -28,29 +28,39 @@ On success, an object with a "channels" key is returned containing a list of 0
 or more objects.
 
 Each object in the list contains the following data:
-- 'source' - The node that initiated the channel creation
-- 'destination' - The node that was the target of the channel creation
-- 'short_channel_id' - The channel identifier
-- 'public' - Boolean value, is publicly available
-- 'satoshis' - 
-- 'message_flags' - 
-- 'channel_flags' - 
-- 'active' - 					
-- 'last_update' -
-- 'base_fee_millisatoshi' -
-- 'fee_per_millionth' -
-- 'delay' -
-
-If 'shcort_channel_id' is supplied and no matching channels are found, a
+
+- 'source' : The node that initiated the channel creation
```

Comment:
This is not really true, the source and destination are defined by the direction in which we'd want to traverse the channel. We can't determine who the funder is. The source is the node that specified the fees and CLTV deltas if we want to traverse the channel in this direction.

This is also why we could get 2 results for the 2 different directions when specifying a `short_channel_id`.